package.json

```diff
@@ -57,5 +44,13 @@
     "packageName": "int_algolia_sfra",
     "paths": {
         "base": "../storefront-reference-architecture/cartridges/app_storefront_base/"
+    },
+    "dependencies": {
```

Comment:
If we commit the minified files, it's not mandatory to have them as prod dependencies, they could go in devDependencies. Wdyt?